packages/nimble-components/src/accordion/specs/IxD.md

```diff
@@ -0,0 +1,193 @@
+
+<p>
+<kbd><img src="spec-images/Accordion-13.png"></kbd>
+<br>
+<sub>Do this</sub>
+</p>
+<p>
+<kbd><img src="spec-images/Accordion-14.png"></kbd>
+<br>
+<sub>Don't do this</sub>
+</p>
+
+Multiple disclosures can be used in a row for panels of unrelated content, at the client user's discretion and in accordance with component guidelines and product consistency.
+
+<p>
+<kbd><img src="spec-images/Accordion-15.png"></kbd>
+<br>
+<sub>Avoid this</sub>
+</p>
+
+If only one section (relative to the content) is needed, use a disclosure component.
+
+<p>
+<kbd><img src="spec-images/Accordion-16.png"></kbd>
+<br>
+<sub>Disclosure with other content</sub>
+</p>
+
+### Tree & Navigation
+
+With a disclosure/accordion, the heading is a label for a group of content, not content in-and-of itself. With a tree, the content is the list items and all the list items represent data.
+
+| Component  | Content    | Organization |
+| ---------- | ---------- | ------------ |
+| Disclosure | Any        | Group labels |
+| Tree       | List items | Parent/child |
+
+[ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/treeview/examples/treeview-navigation/) recommends avoiding using a tree component for navigation because the `tree` role requires unecessary implementation of complex functionality that is not needed for navigation styled to look like a tree with expandable sections. However, the accordion is not necessarily the correct component to try and replace it with.
+
+| Component  | Content    | Organization |
+| ---------- | ---------- | ------------ |
+| Disclosure | Any        | Group labels |
+| Tree       | List items | Hierarchical |
+| ???        | List items | Group labels |
+
+[**QUESTION:** Maybe a new navigation-specific component?]
+
+## Design
+
+### Interactions & States
+
+Header buttons are used to expand and collapse associated panels of content. The client user can specify which panels (if any) are open by default.
+
+<p>
+<kbd><img src="spec-images/Accordion-1.png"></kbd>
+<br>
+<sub>Default (all panels closed)</sub>
+</p>
+
+Hovering over a header button indicates interactivity.
+
+<p>
+<kbd><img src="spec-images/Accordion-2.png"></kbd>
+<br>
+<sub>Hover</sub>
+</p>
+
+Clicking anywhere on a header button opens (if closed) or closes (if opened) the associated panel of content. [**QUESTION:** Do panels only display block content (one component per "line") or do they display flex content?]
+
+<p>
+<kbd><img src="spec-images/Accordion-3.png"></kbd>
+<br>
+<sub>Click</sub>
+</p>
+
+The client user can specify whether one panel or more than one panel can be opened at a time. If the former, clicking to open a closed panel will close the open panel (if applicable).
+
+<p>
+<kbd><img src="spec-images/Accordion-4.png"></kbd>
+<br>
+<sub>Click (one panel open at a time)</sub>
+</p>
+
+If a component in a panel is in an error state, the error state is indicated on the associated header button when closed (if not also when open).
```

Comment:
You use the phrase "if not also" a few times in the doc and I want to make sure I understand the intent. Does it mean something like:
1. "and maybe also (but just a suggestion)"?
or
2. "but only if not"?
or
3. something else?

I'd love it if we could come up with language that's more precise (e.g. "must" / "should" / "could" like in [this framework](https://en.wikipedia.org/wiki/MoSCoW_method))

Reply:
You're right, I felt a bit weird about that phrasing as well. Great suggestion, thanks for the reference.